Sources/SwiftCurrent/SwiftCurrent.docc/Tutorials/SwiftCurrent_SwiftUI.tutorial

```diff
@@ -10,30 +10,16 @@
         
         @Steps {
             @Step {
-                Start with Launcher. The Launcher is responsible for calling launch on and passing the launchArgs to the workflow. 
-
+                Start with Launcher.
```

Comment:
Why did these steps get less specific?

Reply:
I think this is part of Richard's reverting of my last commit. Pretty sure my sim links broke the pod linting.

Reply:
Morgan is correct.  Just part of the reverted commit.